packages/teams-js/test/private/meetingRoom.spec.ts

```diff
@@ -33,284 +30,325 @@ describe('meetingRoom', () => {
-      expect(returnedMeetingRoomInfo.clientType).toBe(meetingRoomInfo.clientType);
-      expect(returnedMeetingRoomInfo.clientVersion).toBe(meetingRoomInfo.clientVersion);
-    });
-
-    it('pass sdkError while get meeting room info on mobile', async () => {
-      await framelessPlatformMock.initializeWithContext('content');
-
-      const promise = meetingRoom.getPairedMeetingRoomInfo();
-
-      const message = framelessPlatformMock.findMessageByFunc('meetingRoom.getPairedMeetingRoomInfo');
-      expect(message).not.toBeNull();
-      expect(message.args.length).toBe(0);
-
-      const callbackId = message.id;
-      framelessPlatformMock.respondToMessage({
-        data: {
-          id: callbackId,
-          args: [{ errorCode: 500, message: errorMessage }],
-        },
-      } as DOMMessageEvent);
-
-      return expect(promise).rejects.toEqual({ errorCode: 500, message: errorMessage });
+    it('meetingRoom.sendCommandToPairedMeetingRoom should not allow calls before initialization', () => {
+      expect(() => meetingRoom.sendCommandToPairedMeetingRoom('mute')).rejects.toThrowError(
+        new Error(errorLibraryNotInitialized),
+      );
     });
 
-    it('should allow getPairedMeetingRoomInfo calls on desktop', async () => {
-      await framedPlatformMock.initializeWithContext('content');
-
-      meetingRoom.getPairedMeetingRoomInfo();
-
-      const message = framedPlatformMock.findMessageByFunc('meetingRoom.getPairedMeetingRoomInfo');
-      expect(message).not.toBeNull();
-      expect(message.args.length).toBe(0);
+    it('meetingRoom.registerMeetingRoomCapabilitiesUpdateHandler should not allow calls before initialization ', () => {
+      expect(() => meetingRoom.registerMeetingRoomCapabilitiesUpdateHandler(emptyHandler)).toThrowError(
+        new Error(errorLibraryNotInitialized),
+      );
     });
-  });
 
-  describe('sendCommandToPairedMeetingRoom', () => {
-    it('should not allow calls before initialization', () => {
-      return expect(meetingRoom.sendCommandToPairedMeetingRoom('mute')).rejects.toThrowError(
+    it('meetingRoom.registerMeetingRoomStatesUpdateHandler should not allow calls before initialization ', () => {
+      expect(() => meetingRoom.registerMeetingRoomStatesUpdateHandler(emptyHandler)).toThrowError(
         new Error(errorLibraryNotInitialized),
       );
     });
+  });
 
-    it('sendCommandToPairedMeetingRoom should throw error when meetingRoom is not supported.', async () => {
-      await framelessPlatformMock.initializeWithContext('content');
-      framedPlatformMock.setRuntimeConfig({ apiVersion: 1, supports: {} });
-      expect(meetingRoom.sendCommandToPairedMeetingRoom('mute')).rejects.toEqual(errorNotSupportedOnPlatform);
+  describe('frameless', () => {
+    let utils: Utils = new Utils();
+    beforeEach(() => {
+      utils = new Utils();
+      utils.mockWindow.parent = undefined;
+      utils.messages = [];
+      GlobalVars.isFramelessWindow = false;
```

Comment:
this is a stupid question but does this get flipped to true for frameless or no

Reply:
Yes it does.